python/sunbeam/schedule.py

```diff
@@ -16,6 +16,22 @@ def wells(self):
     def group(self, name):
         return Group(self._group(name), self)
 
+    def group_tree(self, timestep):
```

Comment:
I'm wondering if this shouldn't be a property returning a Sequence.  That way, I could write
`tree[0]`, `tree[1]` etc...